tests/rbd/rbd_clonev2.py

```diff
@@ -33,37 +95,22 @@ def run(**kw):
     4. Create clone for an image which is having snapshots
     5. Remove parent image snapshot and check the status
     6. Flattened the clone and check the trash views
+    7. Repeat the above steps for ecpool
     """
     log.info("Running snap and clone operations with v2clone function")
     log.info("executing *****")
-    rbd = Rbd(**kw)
-    pool = rbd.random_string()
-    image = rbd.random_string()
-    snap = rbd.random_string()
-    clone = rbd.random_string()
-    size = "1G"
+    if kw.get("config"):
+        kw["config"]["ec-pool-k-m"] = "use_default"
+    else:
+        kw = {"config": {"ec-pool-k-m": "use_default"}}
+    rbd_obj = initial_rbd_config(**kw)
+    rc = 1
+    if rbd_obj:
+        rc = rbd_clonev2(rbd_obj.get("rbd_reppool"), "rep_pool_config", **kw)
```

Comment:
can we have some info messages saying test running on **EC pools** and **Replicated pools**?

Reply:
Done.